syncstorage-postgres/src/db/batch_impl.rs

```diff
@@ -142,7 +141,40 @@ impl BatchDb for PgDb {
             })
             .await?;
         let default_ttl_seconds = DEFAULT_BSO_TTL as i64;
+        let ts_datetime = timestamp.as_datetime()?;
 
+        sql_query(
+            "UPDATE bsos
+             SET
+                 sortindex = COALESCE(src.sortindex, bsos.sortindex),
+                 payload = COALESCE(src.payload, bsos.payload),
+                 modified = $3::TIMESTAMP,
+                 expiry = COALESCE(
+                     CASE
+                         WHEN src.ttl IS NOT NULL THEN $3::TIMESTAMP + (src.ttl || ' seconds')::INTERVAL
```

Comment:
btw do we need any of the casts in either of these queries other than `INTERVAL`?

```suggestion
                 modified = $3,
                 expiry = COALESCE(
                     CASE
                         WHEN src.ttl IS NOT NULL THEN $3 + (src.ttl || ' seconds')::INTERVAL
```

Reply:
🤦 Good catch on the `TIMESTAMP`.  I'll double check the casts.  Thanks

Reply:
np, I should have fixed it myself w/ the `TIMESTAMPTZ` change